cmd/kots/cli/admin-console-generate-manifests.go

```diff
@@ -77,5 +100,7 @@ func AdminGenerateManifestsCmd() *cobra.Command {
 	cmd.Flags().Bool("minimal-rbac", false, "set to true to use the namespaced role and bindings instead of cluster-level permissions")
 	cmd.Flags().StringSlice("additional-namespaces", []string{}, "Comma separate list to specify additional namespace(s) managed by KOTS outside where it is to be deployed. Ignored without with '--minimal-rbac=true'")
 
+	registryFlags(cmd.Flags())
```

Comment:
added registry flags to `generate-manifests` so that we can test it with ttl images